src/styles/styles.scss

```diff
@@ -1,7 +1,9 @@
 $color_address: #C0CACC;
```

Comment:
Rather than add constants at the top of the styles.scss file.  Would it be possible for us to create a `constants.scss` file that we imported in the appropriate places?

I think we could also create a `styles` barrel so we could import all of these style files in our components.

Reply:
I added a few additional files `_constants.scss` and `_spacing.scss` for the top level variables in the styles.scss file.  I recommend we refrain from making styles that refer to elements (`h1`) moving forward.